ffs/ffs_terms.md

```diff
@@ -0,0 +1,172 @@
+
+#### (a) Activity Reporting
+
+The Contractor must provide at least one status report about their progress each month. This will usually be done in a bi-monthly dev or community meeting (whichever is most appropriate for the proposal) on the `#monero-dev` or `#monero-community` IRC channels respectively.
+
+The Contractor should aim to attend all meetings if possible. Should the Contractor wish to provide more updates, this can be done on the [/r/Monero](https://www.reddit.com/r/Monero/) subreddit with title prefix [FFS UPDATE] or on the comment section of the proposal thread.
+
+#### (b) Resignation/Abandonment
+
+If the Contractor does not wish to complete a proposal for any reason, he/she may resign from the project. It is requested that the Contractor formally inform the community about his/her decision in one of the bi-monthly dev or community meetings on `#monero-dev` or `#monero-community` IRC channels. The Contractor may claim any fairly completed milestones after leaving.
+
+If the Contractor fails to make two consecutive monthly status reports, the funded proposal would be considered abandoned.
+
+If the Contractor leaves the the project or the project is deemed abandoned, the proposal can be picked up by another contractor for completion or, at the discretion of the Core Team, the proposal maybe relegated back to the Idea stage and raised funds re-appropriated to the General Fund.
+
+#### (c) Amendments
+
+If a funded proposal needs to be amended for any reason, the Contractor, or a member of the Core Team, must post a `Request to Amend` in the proposal’s thread.
+
+To approve or reject the change, Donors that represent at least 51% of the value of the contributed amount of the proposal must type either “YEA” or “NAY” on the proposal’s thread and include the `TXID` and `TXKEY` to prove their contributed amount.
+
+If quorum is not reached within four weeks from the date the `Request to Amend` is made, the Contractor may continue working on the original proposal or, at the discretion of the Core Team, the proposal maybe relegated back to the Idea stage and raised funds re-appropriated to the General Fund.
+
+##  5. Terms & Conditions
+
+Now for the legal stuff, by submitting a FFS proposal, the Donor and Contractor duly agree that they have read and understood the FFS process described above and accept the following terms & conditions in full.
+
+#### (a) Licensing
+
+The completed deliverable shall be under [a BSD-3 license](https://opensource.org/licenses/BSD-3-Clause).
+
+#### (b) Code of Conduct & Vulnerability Response Process
+
+The Contractor agrees to adhere to [Code of Conduct (22/C4.1)](https://rfc.zeromq.org/spec:22/C4/) and the [Monero Project Vulnerability Response Process](https://github.com/anonimal/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md).
```

Comment:
>The Contractor agrees to adhere to [Code of Conduct (22/C4.1)](https://rfc.zeromq.org/spec:22/C4/)

This should link [here](https://github.com/monero-project/monero/blob/master/CONTRIBUTING.md#code-of-conduct-22c41) instead.